tt_metal/hw/inc/blackhole/noc_nonblocking_api.h

```diff
@@ -7,466 +7,341 @@
 #include <stdint.h>
 
 #include "noc_parameters.h"
-#include "risc_attribs.h"
-
-#ifdef RISC_B0_HW
-const uint32_t NCRISC_WR_CMD_BUF = 3;
-const uint32_t NCRISC_WR_CMD_BUF_0 = 0;
-const uint32_t NCRISC_WR_CMD_BUF_1 = 1;
-const uint32_t NCRISC_SMALL_TXN_CMD_BUF = 3;
-#else
-const uint32_t NCRISC_WR_CMD_BUF = 0;
-const uint32_t NCRISC_WR_CMD_BUF_0 = 0;
-const uint32_t NCRISC_WR_CMD_BUF_1 = 1;
-const uint32_t NCRISC_WR_CMD_BUF_2 = 2;
-const uint32_t NCRISC_SMALL_TXN_CMD_BUF = 3;
-#endif
-
-const uint32_t NCRISC_WR_DEF_TRID = 0;
-const uint32_t NCRISC_WR_LOCAL_TRID = 1;
-const uint32_t NCRISC_RD_DEF_TRID = 2;
-const uint32_t NCRISC_HEADER_RD_TRID = 3;
-const uint32_t NCRISC_RD_START_TRID = 4;
-const uint32_t NCRISC_RD_END_TRID = 13;
-const uint32_t NCRISC_ETH_START_TRID = 14;
-const uint32_t NCRISC_ETH_END_TRID = 15;
+
+////
+/*TODO: RT review this file, currently using wormhole b0 copy, check if any changes needed for BH*/
```

Comment:
In #8704 I enumerated some of the changes we need to make here. I can go through that in a sep PR and we can merge what you have in here to unblock the compilation